types/instance.d.ts

```diff
@@ -57,7 +57,7 @@ export interface FastifyInstance<
    * This method is now deprecated and will throw a `FST_ERR_MISSING_MIDDLEWARE` error.
```

Comment:
This comment is false. It's not deprecated and you'd need to use https://github.com/fastify/middie or https://github.com/fastify/fastiy-express.

Reply:
Okidok, I'll update it. Then would you prefer I describe the arguments in full (instead of `unknown[]`)?

Reply:
Yes please!

Reply:
https://github.com/fastify/fastify/blob/7ddbef9d0f2234b506cb1a3849c027f72f1c9c04/fastify.js#L392

Is this comment still false?

Reply:
I left out the `@throws FST_ERR_MISSING_MIDDLEWARE` to reduce the risk of that bit of documentation getting outdated. If anything I'd rather remove the comment entirely or have it point to the docs instead.

Reply:
But to be clear, @Ethan-Arrowood, would you rather I add back the note about the error?

Reply:
Doh. Nevermind, I think I get it now.